enodi/Inventory
# #156619087 Display all Items

#### What does this PR do?
This PR displays all items from the database on the table on the dashboard page

#### Description of Task to be completed?
- Ensure all items in the database are displayed on the table

#### How should this be manually tested?
-Log into the application
- On successful sign in, you will be redirected to the dashboard page.
- On the dashboard page, you notice that all items are displayed on the table.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID: #156619087](https://www.pivotaltracker.com/n/projects/2146717/stories/156619087)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29783644/38510782-f2ecad20-3c1d-11e8-8408-ac514cacd22d.png)

![image](https://user-images.githubusercontent.com/29783644/38510795-fa8be208-3c1d-11e8-9436-96d1b7170a9b.png)
